companies/sbahn-berlin.json

```diff
@@ -0,0 +1,23 @@
+{
+    "slug": "sbahn-berlin",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "S-Bahn Berlin GmbH",
+    "address": "Elisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
+    "phone": "+49 30 29743333",
+    "email": "Datenschutz.Regio@deutschebahn.com",
```

Comment:
Since this is a data protection-specific email, please set `suggested-transport-medium` to `email`.